src/compiler/types.ts

```diff
@@ -2447,6 +2449,22 @@ namespace ts {
         [option: string]: string | number | boolean;
     }
 
+    export interface TypingOptions {
+        enableAutoDiscovery?: boolean;
+        include?: string[];
+        exclude?: string[];
+        [option: string]: string[] | boolean;
+    }
+
+    export interface DiscoverTypingsInfo {

```

Comment:
Is this type originally intended to be used as the parameter type for `discoverTypings`?